.gitignore

```diff
@@ -1,4 +1,4 @@
 vendor/
 composer.lock
 tests/FacebookTestCredentials.php
-
+.idea/
```

Comment:
must be removed, this should be part of a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), it's not related to the project